lm_eval/tasks/belebele/belebele_acm_Arab.yaml

```diff
@@ -0,0 +1,4 @@
+"dataset_name": "acm_Arab"
+"description": "A split of Belebele for the acm_Arab language.\n\n"
```

Comment:
So sorry for the confusion here,

`description` is meant to be the a string of text that is prepended to the whole prompt. Something like "The following are multiple choice tasks for X topic". Not meant to be a description for the user.

Reply:
Ok fixed